lbz/configuration.py

```diff
@@ -0,0 +1,60 @@
+from abc import ABCMeta, abstractmethod
+from os import getenv
+from typing import Any, Callable, Generic, Optional, TypeVar
+
+from lbz.aws_ssm import SSM
+from lbz.exceptions import ConfigurationMissingError, ConfigurationParsingError
+
+T = TypeVar("T")
+
+
+class ConfigValue(Generic[T], metaclass=ABCMeta):
+    """Mandatory Configuration
+
+    This class is not supporting None as outcome value.
+    """
+
+    def __init__(
+        self,
+        key: str,
+        parser: Callable[[str], T] = str,  # type: ignore
```

Comment:
Once we changed `getter()` to return `Any`, we need to adjust the parser as well:
```suggestion
        parser: Callable[[Any], T] = str,  # type: ignore
```